src/style/app.module.css

```diff
@@ -1361,11 +1418,10 @@ hr {
   padding: 15px;
   padding-bottom: 0px;
   width: 33%;
-
+  border: 1px solid var(--input-shadow);
   box-sizing: border-box;
   background: var(--white-color);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`